feature/settings/src/main/res/values/strings.xml

```diff
@@ -48,4 +48,15 @@
     <string name="flash_mode_status_on">Flash is On</string>
     <string name="flash_mode_status_off">Flash is Off</string>
 
+    <!-- Aspect Ratio  setting strings -->
+    <string name="aspect_ratio_title">Set Aspect Ratio</string>
+
+    <string name="aspect_ratio_selector_9_16">9:16</string>
+    <string name="aspect_ratio_selector_3_4">3:4</string>
+    <string name="aspect_ratio_selector_1_1">1:1</string>
+
+    <string name="aspect_ratio_description_9_16">Aspect Ratio is 9:16</string>
+    <string name="aspect_ratio_description_3_4">Aspect Ratio is 3:4</string>
+    <string name="aspect_ratio_description_1_1">Aspect Ratio is 1:1</string>
```

Comment:
How do these read out loud? We should make sure the accessibility actually says "nine to sixteen"

Reply:
this is actually just the description text that is visible.
I actually still need to update the composable parameters to take in accessibility text for ALL settings. 
I'll put this into a todo and write a bug for that.